src/tyr/serializers.cc

```diff
@@ -83,10 +83,29 @@ std::vector<std::string> openlr_edges(const TripLeg& leg) {
 } // namespace
 namespace valhalla {
 namespace tyr {
-std::string serializeStatus(const Api&) {
-  // TODO: once we decide on what's in the status message we'll fill out the proto message in
-  // loki/thor/odin and we'll serialize it here
-  return "{}";
+std::string serializeStatus(const Api& request) {
+  static auto actions_array = json::array({});
+  if (actions_array->empty()) {
+    istringstream iss(request.status().actions());
+    std::vector<std::string> actions((istream_iterator<string>(iss)), istream_iterator<string>());
+    for (const auto& action : actions) {
+      actions_array->push_back(action);
+    }
+  }
+
+  auto status_msg = baldr::json::map({{"version", std::to_string(VALHALLA_VERSION_MAJOR) + "." +
+                                                      std::to_string(VALHALLA_VERSION_MINOR) + "." +
+                                                      std::to_string(VALHALLA_VERSION_PATCH)},
+                                      {"has_tiles", request.status().has_tiles()},
+                                      {"has_admins", request.status().has_admins()},
+                                      {"has_timezones", request.status().has_timezones()},
+                                      {"has_live_traffic", request.status().has_live_traffic()},
+                                      {"bbox", request.status().bbox()},
```

Comment:
the bbox is a stringified JSON now.. would be much nicer to have it as JSON object, but that'd require to move a little code around in `connectivity_map.cc` to support just getting the coords of the different regions, put the coords as a `Ring` to the PBF in loki's `status_action.cc` and build the GeoJSON object here. While doing that we could also consider using boost:geometry in `connectivity_map.cc` to simplify the polgons a little, they're slightly overly verbose atm.

Inclined to leave it as is for now and visit again later?

Reply:
forget it.. using rapidjson instead to serialize the response is much easier to make `bbox` a JSON object. kinda lazy way putting a (de)serialize JSON inside a PBF (de)serializer instead of using pbf objects right away. other option could be to do the above suggestion, more code changes though.